resources/cloudformation-stack.go

```diff
@@ -44,9 +46,42 @@ type CloudFormationStack struct {
 }
 
 func (cfs *CloudFormationStack) Remove() error {
-	_, err := cfs.svc.DeleteStack(&cloudformation.DeleteStackInput{
+	stack, err := cfs.svc.DescribeStacks(&cloudformation.DescribeStacksInput{
```

Comment:
There is already an up-to-date object in `cfs.stack`.

Reply:
true. done. 
i also restructured the code a bit.

Reply:
@svenwltr we actually need to get a fresh version of the stack object at this place. otherwise it will run into a race-condition where `DELETION_FAILED` is not set while it actually should be.

Reply:
Ok, *aws-nuke* runs the re-scan after it tried to remove the resource. This means it gets refreshed after `Remove()` was called.